### ISSUE TITLE
Make uncommenter

### ISSUE THREAD

**Issue by almiche** [jusleg/quotifier#1]
# It's local on the the remote server - Soulja Boy

**Comment by jusleg** [comment]
Good idea 💡. It is easy to implement but it would also remove every comment the user wrote. Our quotes should have a unique characteristic so we only delete them

**Comment by almiche** [comment]
I like that idea. Go bing or go home ! -Triple H

**Comment by kevcamel** [comment]
boringify.py done, but we still have the issue c3n@ spoke about
